src/composables/downloader.js

```diff
@@ -1,7 +1,7 @@
 import { computed, ref } from 'vue'
 import { invoke } from '@tauri-apps/api/core'
 
-const delay = (time) => new Promise((resolve, reject) => setTimeout(resolve, time))
+const delay = (time) => new Promise((reso, rejectlve, reject) => setTimeout(resolve, time))
```

Comment:
It seems there is a typo here.

```suggestion
const delay = (time) => new Promise((resolve, reject) => setTimeout(resolve, time))
```